bears/php/PHPCodeSnifferBear.py

```diff
@@ -26,7 +26,7 @@ class PHPCodeSnifferBear:
             [DistributionRequirement(apt_get='php-codesniffer',
                                      zypper='php-pear-php_codesniffer',
                                      ),
-             ComposerRequirement('squizlabs/php_codesniffer'),
+             ComposerRequirement('php_codesniffer'),
```

Comment:
And this one should be `ComposeRequirement('squizlabs/php_codesniffer', '3.3.2')
(you can find the latest version by searching the composer package list (hosted at packagist.org) -> https://packagist.org/packages/squizlabs/php_codesniffer)

Reply:
@Makman2  ```~``` operator will automatically take on the required version.